src/components/Internationalize.js

```diff
@@ -5,22 +5,21 @@
  */
 
 import React, { Children } from 'react';
-import { IntlProvider, addLocaleData } from 'react-intl';
+import { IntlProvider } from 'react-intl';
 
 import i18n from '../utils/i18n';
 
 type Props = {
     children?: any,
+    messages: Object,
 };
 
-addLocaleData(i18n.localeData);
-
-const Internationalize = ({ children }: Props) => {
-    const { language, messages } = i18n;
+const Internationalize = ({ children, messages }: Props) => {
+    const { getLocale, language } = i18n;
```

Comment:
It seems like the module should be destructured at import time, since it's not a prop?